src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/Frame.cs

```diff
@@ -1267,6 +1267,25 @@ protected void ReportApplicationError(Exception ex)
             Log.ApplicationError(ConnectionId, ex);
         }
 
+        public void Tick()
+        {
+            if (_requestProcessingStatus == RequestProcessingStatus.RequestPending && // we're in between requests and
```

Comment:
We almost never put comments at the end of a line. I get why you did it, but just put the comment above the if and use "and" in the comment.